volatility3/framework/plugins/windows/handles.py

```diff
@@ -25,7 +25,7 @@ class Handles(interfaces.plugins.PluginInterface):
     """Lists process open handles."""
 
     _required_framework_version = (2, 0, 0)
-    _version = (1, 0, 1)
```

Comment:
Hmmmm, this is a tricky one.  It does technically alter the output of `find_sar_value`, but that isn't directly expose, so I think this does only class as an internal change and so the version number you've chosen looks right.